source/helpers/index.js

```diff
@@ -8,6 +9,7 @@ import * as ValidationHelper from './ValidationHelper';
 
 export {
   ApiRequest,
+  DataStructure,
```

Comment:
The name DataStructure feels very general. And perhaps it should be, it's just hard to tell what DataStructure does as a helper. I don't have any suggestions at this moment but i would think that a better name could be found :)